rpmdeplint.fmf

```diff
@@ -21,5 +28,5 @@ prepare:
         fi
 
 execute:
-    how: shell
-    script: /rpmdeplint/run_rpmdeplint.py --release $RELEASE_ID --task-id $TASK_ID
+    /rpmdeplint:
```

Comment:
This extra line does not have any effect and could/should be removed.

Reply:
Fixed, thanks :wink: